wallet/wallet.go

```diff
@@ -2,16 +2,23 @@ package wallet
 
 import (
```

Comment:
Would be nice to have an integration test that starts with a single large output (e.g. 100SC) and splits it into 10 times 10SC or something like that. Just to verify that the number of outputs increased. Maybe followed by another call that fails due to insufficient funds and another one that reduces the number of outputs again.

Reply:
I ended up adding a unit test. We definitely need a test miner so we can have integration tests for this type of thing, also contract formation/renewals. I don't mind adding that but I figured it'd be best to do it in a separate PR, the unit test covers the functionality well enough I feel.

Reply:
You can grab the one I built to test the host, it's very lightweight: https://github.com/SiaFoundation/hostd/blob/master/internal/cpuminer/miner.go